tests/config/pytest_modules.yml

```diff
@@ -3368,6 +3368,18 @@ sourmash/taxannotate:
   - modules/nf-core/sourmash/taxannotate/**
   - tests/modules/nf-core/sourmash/taxannotate/**
 
+spaceranger/count:
```

Comment:
for these module tests where you build the reference (`mkgtf`, `mkref`, then `count`), do you actually need to add the `spaceranger/mkgtf` and `spaceranger/mkref` lines here, or am I a naïve rube and unnecessarily added that for `cellranger`? 😅

Reply:
You mean that `mkgtf` and `mkref` are tested implicitly by the `count` workflow anyway? 

I think it makes sense to keep then anyway (in case only `mkref`/`mkgtf` were to be modified), these lines were automatically added by `nf-core modules create` and yes, they are there fore cellranger as well. 

Btw, they are there multiple times 😜 

https://github.com/nf-core/modules/blob/fa58fa05516e77023f6a8779d1cecbdceeca113b/tests/config/pytest_modules.yml#L635-L667

Reply:
yes, I guess that I wasn't clear, I didn't realize that you could implicitly test `mkref` and `mkgtf`, that's why I always added them as explicit tests for GEX and VDJ modules

anyways this doesn't sound like a problem, so I'll resolve 👍🏼